gum-gum-htb.js

```diff
@@ -283,15 +326,15 @@ function GumGumHtb(configs) {
              * these local variables */
 
             /* the bid price for the given slot */
-            var bidPrice = curBid.price;
+            var bidPrice = curBid.ad.price;
 
             /* the size of the given slot */
-            var bidSize = [Number(curBid.width), Number(curBid.height)];
+            var bidSize = [Number(curBid.ad.width), Number(curBid.ad.height)];
```

Comment:
Same as above

Reply:
Should be taken care by the default values